Content.Server/Station/Systems/StationSpawningSystem.cs

```diff
@@ -182,10 +182,13 @@ public void EquipIdCard(EntityUid entity, string characterName, JobPrototype job
         if (!InventorySystem.TryGetSlotEntity(entity, "id", out var idUid))
             return;
 
-        if (!EntityManager.TryGetComponent(idUid, out PdaComponent? pdaComponent) || !TryComp<IdCardComponent>(pdaComponent.ContainedId, out var card))
+        var cardId = idUid.Value;
+        if (TryComp(idUid, out PdaComponent? pdaComponent) && pdaComponent.ContainedId != null)
```

Comment:
`TryComp<PdaComponent>(idUid, out var pdaComponent)` :trollface:

Reply:
The original is the correct one.

Reply:
noob